lib/ex_aws/kms_client_encryption.ex

```diff
@@ -0,0 +1,52 @@
+defmodule ExAws.KMS.ClientEncryption.Result do
+  @enforce_keys [:encrypted_data_key, :ciphertext]
+  defstruct [:encrypted_data_key, :ciphertext]
+end
+
+defmodule ExAws.KMS.ClientEncryption do
```

Comment:
This module needs documentation showing what example usage is like. I'm honestly not sure what role this module fills.